desktop/ui/src/main/java/org/datacleaner/panels/result/AnalyzerResultPanel.java

```diff
@@ -99,6 +100,7 @@ public AnalyzerResultPanel(RendererFactory rendererFactory, ProgressInformationP
     public void setResult(final AnalyzerResult result) {
         final String headerText = getHeaderText();
         _progressInformationPanel.addUserLog("Rendering result for " + headerText);
+        _headerLabel.setText(LabelUtils.getLabel(_componentJob, result, false, false, false));
```

Comment:
Yes ComponentJob is, but AnalyzerResult is not. The HasLabelAdvice interface is implemented by the AnalyzerResult, not the ComponentJob. So first I set the label without the result using the logic that was already there, and later I update it with the analyzerresult (if the result has the interface, otherwise it is just the same).